meshmode/mesh/__init__.py

```diff
@@ -848,6 +857,16 @@ class Mesh(Record):
 
     .. attribute:: nelements
 
+    .. attribute:: element_nr_base
+
+        An array of size ``(len(groups),)`` of starting element indices for
+        each group in the mesh.
+
+    .. attribute:: node_nr_base
```

Comment:
```suggestion
    .. attribute:: base_node_nrs
```
? (to convey plural and avoid confusion with existing attribute)

Reply:
Done.